### ISSUE TITLE
ListBox component accessibility issues

### ISSUE THREAD

**Issue by andrey-post** [primefaces/primeng#16152]
### Describe the bug

`ListBox` component accessibility violations:
1. Scrollable region must have keyboard access
2. ARIA hidden element must not be focusable or contain focusable elements

![image](https://github.com/user-attachments/assets/5582a48f-a8b1-472b-a2b8-4c1951d04886)
![image](https://github.com/user-attachments/assets/980a4091-eee5-4d82-bb31-790a8b0bafa0)

### Environment

Latest Chrome on Windows 10, Angular CLI project with Angular@17.3.12 and PrimeNG@17.18.6

### Angular version

17.3.12

### PrimeNG version

17.18.6

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

18.20.2

### Steps to reproduce the behavior

1. Open https://primeng.org/listbox
2. Scan with Axe Dev Tools accessibility scanner

### Expected behavior

No critical or serious accessibility issues coming from the inside of the component.
